plugins/statistics_exporter/instance.ts

```diff
@@ -1,7 +1,23 @@
-"use strict";
-const lib = require("@clusterio/lib");
+import * as lib from "@clusterio/lib";
 const { Gauge } = lib;
 
+type IpcStats = {
+	game_tick: number,
+	player_count: number,
+	game_flow_statistics: {
+		pollution_statistics: IpcFlowType,
+	},
+	force_flow_statistics: {
+		[key:string]: {
+			[key:string]: IpcFlowType,
+		}
+	},
+}
+
+type IpcFlowType = {
+	"input": number,
+	"output": number,
+}
```

Comment:
The shape of the "input" and "output" entries is an object where string keys map to numbers. i.e. `"input": Record<string, number>`.